protos/varfish/v1/query_seqvars.proto

```diff
@@ -0,0 +1,247 @@
+    TranscriptVariant = 47;
+    UpstreamGeneVariant = 48;
+}
+
+// Enumeration for recessive mode queries.
+enum RecesssiveMode {
+    Recessive = 0;
+    CompoundRecessive = 1;
+}
+
+// Choices for failing quality thresholds on genotypes.
+enum FailChoice {
+    // Ignore failure
+    Ignore = 0;
+    // Drop whole variant
+    Drop = 1;
+    // TODO COMMENT
+    NoCall = 2;
+}
+
+// Choice for genotype.
+enum GenotypeChoice {
+    // Any genotype
+    Any = 0;
+    // Ref. genotype
+    Ref = 1;
+    // Het. genotype
+    Het = 2;
+    // Hom. genotype
+    Hom = 3;
+    // Non-hom genotype
+    NonHom = 4;
+    // Variant genotype
+    Variant = 5;
+    // Index in comp. het. recessive inheritance
+    ComphetIndex = 6;
+    // Index in recessive inheritance
+    RecessiveIndex = 7;
+    // Parent in recessive inheritance
+    RecessiveParent = 8;
+}
+
+// Quality settings for one sample.
+message QualitySettings {
+    // Minimal coverage for het. sites
+    optional int32 dp_het = 1;
+    // Minimal coverage for hom. sites
+    optional int32 dp_hom = 2;
+    // Minimal genotype quality
+    optional int32 gq = 3;
+    // Minimal allele balance for het. variants
+    optional float ab = 4;
+    // Minimal number of alternate reads
+    optional int32 ad = 5;
+    // Maximal number of alternate reads
+    optional int32 ad_max = 6;
+    // Behaviour on failing quality thresholds
+    FailChoice fail = 7;
+}
+
+message Range {
+    int32 start = 1;
+    int32 end = 2;
+}
+
+message GenomicRegion {
+    // Chromosome
+    string chrom = 1;
+    // Range of region
+    optional Range range = 2;
+}
+
+message ClinVarOptions {
+    // Wether to require ClinVar membership
+    bool require_in_clinvar = 1;
+    // Whether to include benign ClinVar variants
+    bool include_benign = 2;
+    // Whether to include pathogenic ClinVar variants
+    bool include_pathogenic = 3;
+    // Whether to include likely benign ClinVar variants
+    bool include_likely_benign = 4;
+    // Whether to include likely pathogenic ClinVar variants
+    bool include_likely_pathogenic = 5;
+    // Whether to include uncertain significance ClinVar variants
+    bool include_uncertain_significance = 6;
+    // Whether to include conflicting interpretation ClinVar variants
+    bool include_conflicting_classifications = 7;
+}
+
+message InhouseFrequencyOptions {
+    // Whether to enable filtration by 1000 Genomes.
+    bool enabled = 1;
+    // Maximal number of in-house carriers
+    optional int32 carriers = 2;
+    // Maximal number of in-house heterozygous carriers
+    optional int32 heterozygous = 3;
+    // Maximal number of in-house homozygous carriers
+    optional int32 homozygous = 4;
+    // Maximal number of in-house hemizygous carriers
+    optional int32 hemizygous = 5;
+}
+
+// TODO: Might be better to structure with an optional submessage instead of all these individual optionals unless they need to be this way for some reason
+
+message GnomadOptions {
```

Comment:
Should be called and adjusted, see change below.

```
message GnomadNuclearOptions {
    // Whether to enable filtration by 1000 Genomes.
    bool enabled = 1;
    // Maximal number of carriers
    optional int32 carriers = 2;
    // Maximal number of heterozygous carriers
    optional int32 heterozygous = 3;
    // Maximal number of homozygous carriers
    optional int32 homozygous = 4;
    // Maximal number of hemizygous carriers
    optional int32 hemizygous = 5;
}
```

Also, probably add here:

```
message GnomadMitchondrialOptions {
    // Whether to enable filtration by 1000 Genomes.
    bool enabled = 1;
    // Maximal number of carriers
    optional int32 carriers = 2;
    // Maximal number of heteroplasmic carriers.
    optional int32 heteroplasmic = 3;
    // Maximal number of homoplasmic carriers.
    optional int32 homoplasmic = 4;
    // Maximal allele frequency.
    optional float allele_frequency = 5;
}
```

Reply:
Is that missing gnomad_exomes.frequency? (freqency.rs for example wants to access `exomes_frequency` but can't)

Reply:
Yes, `GnomadNuclearOptions` must have

```
    // Maximal allele frequency.
    optional float allele_frequency = 6;
```